src/2048-expert/2048-expert.css

```diff
@@ -62,6 +62,14 @@ h1 {
    line-height: 1.5;
 }
 
+#new-btn{
```

Comment:
Might be good to make the button change colour when you hover over it. Some padding on it would also be nice. 
![image](https://user-images.githubusercontent.com/78716153/184559031-453e2536-b557-4a18-81fb-3813daf19bd1.png)

Reply:
Yeah, I will change this in a later issue  #42